binding/nodejs/package.json

```diff
@@ -10,7 +10,7 @@
     "build:types": "tsc --declaration --declarationMap --emitDeclarationOnly --outDir ./dist/types",
     "prepack": "npm run build",
     "prepare": "node copy.js",
-    "test": "NODE_OPTIONS=--max-old-space-size=1024 jest --no-cache",
+    "test": "node --max-old-space-size=1024 ./node_modules/.bin/jest --no-cache",
```

Comment:
maybe it would be best to set this as an environment variable in the job instead?